File: DataStructure_Algorithm/LinkedList/ll.py
class Node:
    data = None
    next = None


class Ll:
    head = None

    def __init__(this):
        this.head = Node()

    def __str__(this):
        ll = ""
        temp = this.head
        while temp.data != None:
            ll += str(temp.data)
            temp = temp.next
        return ll

    def inSert(this, *argv):
        temp = this.head
        while temp.next != None:
            temp = temp.next
        for i in argv:
            temp.next = Node()
            temp.data = i
            temp = temp.next


ll = Ll()
ll.inSert([i for i in range(10)])

print(ll)

File: DataStructure_Algorithm/LinkedList/ll.js
class Node {
    data = undefined;
    next = undefined;
}

class Ll {
    head = undefined;
    constructor() {
        this.head = new Node();
    }
    insert(num) {
        let temp = this.head;
        while (temp.data != undefined) {
            temp = temp.next;
        }
        temp.next = new Node();
        temp.data = num;
    }
    show() {
        let str = '';
        let temp = this.head;
        while (temp.data != undefined) {
            str += `${temp.data.toString()} `;
            temp = temp.next;
        }
        console.log(str);
    }
}

let ll = new Ll();
for (let i = 0; i < 10; i++) {
    ll.insert(i);
}

ll.show();